pkgs/applications/networking/instant-messengers/keet/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, appimageTools, fetchurl }:
+
+let
+  pname = "keet";
+  version = "1.2.1";
+
+  src = fetchurl {
+    url = "https://keet.io/downloads/${version}/Keet.AppImage";
+    sha256 = "1f76ccfa16719a24f6d84b88e5ca49fab1c372de309ce74393461903c5c49d98";
+  };
+
+  appimageContents = appimageTools.extract { inherit pname version src; };
+in appimageTools.wrapType2 {
```

Comment:
prefer `rec`
```suggestion
in appimageTools.wrapType2 rec {
  pname = "keet";
  version = "1.2.1";

  src = fetchurl {
    url = "https://keet.io/downloads/${version}/Keet.AppImage";
    hash = "sha256-H3bM+hZxmiT22EuI5cpJ+rHDct4wnOdDk0YZA8XEnZg=";
  };
```

Reply:
This is required for the appimageContents unless we inline it which we normally do not do.